lib/utils/addr.go

```diff
@@ -192,6 +192,14 @@ func MustParseAddrList(aList ...string) []NetAddr {
 	return addrList
 }
 
+func StringAddrsFromNetAddrs(netAddrs []NetAddr) []string {
```

Comment:
Missing Godoc.

Also, instead of making this a standalone function, I would consider implementing a String() method on NetAddrs instead, would be a bit cleaner I think.

```go
type NetAddrs []NetAddr

func (a NetAddrs) String() string {
   ...
}
```

Reply:
It needs to return a slice of strings.
But yeah, a method feels cleaner

Reply:
Not sure I follow this reasoning.  Why refactor a bunch of stuff to use a custom type for just one conversion function?  Unless there is some collection of functionality that we're trying to build around sequences of `NetAddr`s, this seems like unnecessary noise.